actions/ci-publish-npm/action.yml

```diff
@@ -24,18 +24,14 @@ inputs:
     description: "The name of the GitHub release. Defaults to the current branch name"
     required: false
     default: ${{ github.ref_name }}
+  github-release-changelog-path:
```

Comment:
how is this getting used? I only see it as part of the `--notes` flag.

Reply:
That is the only place it is getting used - here is an example: https://github.com/smartcontractkit-test/chainlink-test/releases